tests/functional/pv/pvc_snapshot/test_cephfs_orphaned_snapshot_alert.py

```
@@ -267,87 +281,165 @@ def verify_cephfs_snapshots_bound(self, snap_list_names):
             snap_list_names, constants.CEPHFS_SNAPSHOT_STATE_BOUND
         )
 
+    def _wait_for_orphaned_alert_firing(self):
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `_wait_for_orphaned_alert_firing`

Add return type annotation: `None`

(ANN202)